### PR TITLE
Moving the map to the top right and making smaller

### LINKED ISSUE #233: Moving the map to the top right and making smaller
## Overview
Moving the map to the top right and making smaller

## Required Resources

- [**Note**: List mockups, screenshots, and/or links to design docs required to begin work]
https://docs.google.com/presentation/d/14EWwQ-4xXo9PuZCwDFADdyW6I2vnK4zafvAbuiyankU/edit#slide=id.ge838c51280_0_68
Screen Shot 2021-08-28 at 10 27 08 AM

### PR DESCRIPTION
**Pull request by MarioAP**
Closes #233 

## Description of the Problem / Feature
- The map is large and it covers some of the photos
## Explanation of the solution
- [ ] Making it smaller and moving to the top right

## Instructions on making this work
-

### CODE REVIEW

**Comment by marobo** [comment]
Hau nia sujestaun se bele karik ita la presiza hatudu mapa iha pajina primeiru, ita so bele haree lokasi detail bainhira ita klik iha button detail iha imajen ida!

**Comment by MarioAP** [comment]
> Hau nia sujestaun se bele karik ita la presiza hatudu mapa iha pajina primeiru, ita so bele haree lokasi detail bainhira ita klik iha button detail iha imajen ida!

Hau sei muda fali ba header no uza icon mapa nia deit hanesan dezenu ona iha link nee https://docs.google.com/presentation/d/14EWwQ-4xXo9PuZCwDFADdyW6I2vnK4zafvAbuiyankU/edit#slide=id.ge838c51280_0_68 @marobo

**Comment by PeteCoward** [comment]
Let's get the map expand working before we merge this

File: src/components/Map.vue [hunk 1]
```diff
@@ -53,17 +65,14 @@
         ],
         data() {
             return {
-                expanded: false,
+                isHidden: true,
```
Comment (PeteCoward) [inline src/components/Map.vue:68]: remove this

**Pull request merged**